src/BsConfig.ts

```diff
@@ -106,6 +106,11 @@ export interface BsConfig {
      */
     ignoreErrorCodes?: (number | string)[];
 
+    /**
+     * A map of error codes with their severity level override (error|warn|info)
+     */
+    severityOverride?: Record<number | string, 'error' | 'warn' | 'info' | 'hint'>;
```

Comment:
Let's call this `diagnosticSeverityOverrides` instead? Keeps it more in line with `diagnosticFilters`